src/reynir_correct/pattern.py

```diff
@@ -449,6 +819,15 @@ def create_patterns(cls) -> None:
                     cls.ctx_af,
                 )
             )
+            # Catch "...vegna þess að dýr leita af öðrum smærri dýrum."
+            p.append(
+                (
+                    "af",  # Trigger lemma for this pattern
```

Comment:
A better trigger lemma is 'leita' (it is rarer than 'af')

Reply:
Right, that makes sense